BOOST_ROOT not set correctly anymore

Windows-Standalone-CI for all commits that passed before today

### Steps to reproduce
run windows standalone CI for a commit where it passed

### Expected behavior
should pass where it passed before

### Observed behavior
commit: bf5af4c was green, does only change docu and fails now for windows with
```
  Could NOT find Boost (missing: Boost_INCLUDE_DIR system)
```

### Logfiles
```
Run set     BOOST_ROOT=%BOOST_ROOT_1_72_0%
-- The C compiler identification is MSVC 19.28.29910.0
-- The CXX compiler identification is MSVC 19.28.29910.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.28.29910/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.28.29910/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could NOT find Boost (missing: Boost_INCLUDE_DIR) (Required is at least version "1.69")
CMake Error at C:/Program Files/CMake/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:218 (message):
  Could NOT find Boost (missing: Boost_INCLUDE_DIR system)
```

## Comments
The corresponding environment docu does not state anything differently, but maybe some changes have not yet been documented?
https://github.com/actions/virtual-environments/blob/win19/20201210.0/images/win/Windows2019-Readme.md

--- a/.github/workflows/ci_action_win_standalone.yml
+++ b/.github/workflows/ci_action_win_standalone.yml
@@ -4,6 +4,8 @@ on: [push, pull_request]
 
 env:
   BUILD_TYPE: "Release"
+  BOOST_ROOT: ${{github.workspace}}/3rdparty/boost
+  BOOST_URL: https://sourceforge.net/projects/boost/files/boost/1.72.0/boost_1_72_0.tar.bz2/download
 
 jobs:
   build:
@@ -11,6 +13,26 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+
+    - name: Restore Boost cache
+      uses: actions/cache@v2
+      id: cache-boost
+      with:
+        path: ${{env.BOOST_ROOT}}
+        key: boost
+
+    - name: Install Boost
+      if: steps.cache-boost.outputs.cache-hit != 'true'
+      run: |
+        # fix up paths to be forward slashes consistently
+        BOOST_ROOT=$(echo $BOOST_ROOT | sed 's/\\/\//g')
+        mkdir -p $BOOST_ROOT
+        curl --progress-bar --location --output $BOOST_ROOT/download.tar.bz2 $BOOST_URL
+        7z -o$BOOST_ROOT x $BOOST_ROOT/download.tar.bz2 -y -bd
+        7z -o$BOOST_ROOT x $BOOST_ROOT/download.tar -y -bd
+        cd $BOOST_ROOT && cp -r boost_*/* .
+        rm -rf boost_*/* download.tar.bz2 download.tar
+      shell: bash
 
     - name: Install deps with vcpkg
       shell: bash
@@ -35,7 +57,6 @@ jobs:
       shell: cmd
       working-directory: ${{runner.workspace}}/psen_scan_v2/standalone
       run: |
-        set     BOOST_ROOT=%BOOST_ROOT_1_72_0%
         cmake -G "Visual Studio 16 2019" -A x64 ^
           -DCMAKE_TOOLCHAIN_FILE=%VCPKG_INSTALLATION_ROOT%/scripts/buildsystems/vcpkg.cmake ^
           -S . -B build -DBUILD_TESTING=ON

